### PR TITLE
Drop http sync

### PR DESCRIPTION
**Pull request by janowagner**
This removes wget and curl options for feed download.
These methods always download the entire feed instead of just the delta like rsync does.
The command line options --rsync, --curl and --wget are removed.

### CODE REVIEW

File: tools/greenbone-nvt-sync.in [hunk 1]
```diff
@@ -669,25 +525,21 @@ do_sync ()
...
   echo " --verbose      makes the sync process print details"
   echo " --version      display version"
-  echo " --wget         only use wget to download feed files"
   echo ""
   echo ""
   echo "Environment variables:"
   echo "NVT_DIR         where to extract plugins (absolute path)"
   echo "PRIVATE_SUBDIR  subdirectory of \$NVT_DIR to exclude from synchronization"
   echo "TMPDIR          temporary directory used to download the files"
-  echo "Note that you can use standard ones as well (e.g. http_proxy) for wget/curl"
```
Comment (cfi-gb) [inline tools/greenbone-nvt-sync.in:690]: `echo "Note that you can use standard ones as well (e.g. RSYNC_PROXY) for rsync"`

https://github.com/greenbone/openvas/blob/openvas-scanner-5.0/doc/openvas-nvt-sync.8#L28

**Comment by bjoernricks** [comment]
The comment https://github.com/greenbone/openvas/pull/478#discussion_r407238362 from @cfi-gb should be considered in another pr

File: tools/greenbone-nvt-sync.in (hunk 1 above)
Comment (cfi-gb) [reply tools/greenbone-nvt-sync.in:690]: New PR for this in https://github.com/greenbone/openvas/pull/488

**Comment by cfi-gb** [comment]
Aew PR for the suggestion https://github.com/greenbone/openvas/pull/478#discussion_r407238362 open: #488

**Comment by cfi-gb** [comment]
Official announcement for this change:

https://community.greenbone.net/t/shutting-down-gcf-http-download/5339

**Pull request merged**